### DISCUSSION TITLE
CPP & CTRANSLATE2/PURFVIEW

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8225] [Ideas]
CPP is slower than CTRANSLATE2/PURFVIEW, but cpp transcribe songs in hindi movies, while CTRANSLATE2/PURFVIEW jumped songs in transcription please.

**Comment by Purfview** [comment]
Post the audio with srt file, and post time when it's missing.

Btw, for movies/series you would want to use `--ff_mdx_kim2` setting, but that is only available in `Faster-Whisper-XXL`.

**Reply by Policing2024** [reply]
Faster-Whisper-XXL --model medium "Duty (1986).wav"  --language Hindi --task translate
where to place "--ff_mdx_kim2" in this command prompt please

**Reply by Purfview** [reply]
`Faster-Whisper-XXL --model medium "Duty (1986).wav" --language Hindi --task translate --ff_mdx_kim2`

**Comment by darnn** [comment]
Incidentally, what does Faster-Whisper-XXL actually do that's different from the regular one?

**Reply by Purfview** [reply]
It has some new features not present in the regular one. Like one mentioned above - to extract the voices from the rest of other sounds or background noises.

**Comment by Policing2024** [comment]
how to run
Faster-Whisper-XXL r192.3.3
Direct pls

**Comment by Policing2024** [comment]
How to use Alternative VAD methods?
